src/hooks/useArticlePreviewTracking.js

```diff
@@ -0,0 +1,42 @@
+import { useEffect } from 'preact/hooks'
+import { sendEvent, isInstrumentationEnabled } from 'utils'
+
+const SCHEMA_NAME = 'VirtualPageView'
+const SCHEMA_REV = 19883675
+
+export const useArticlePreviewTracking = (
+  page, source, language
+) => {
+  if (!isInstrumentationEnabled()) {
+    return
+  }
+
+  if (page === undefined || source === undefined) {
+    return
+  }
+
+  const logInukaPageView = () => {
```

Comment:
We should probably update the name of this function :)

Reply:
Yes omg !!! Thanks @neilpquinn